workspace/api/openfdd_bridge/settings.py

```
@@ -0,0 +1,56 @@
+"""Bridge bind, CORS, and LAN defaults."""
+
+from __future__ import annotations
+
+import os
+
+
+def bridge_host() -> str:
+    raw = os.environ.get("OFDD_BRIDGE_HOST", "0.0.0.0").strip()
```

Ruff (0.15.14)

[error] Possible binding to all interfaces

(S104)